arrow/src/array/builder.rs

```diff
@@ -302,6 +302,15 @@ impl BooleanBufferBuilder {
         self.len
     }
 
+    #[inline]
+    pub fn set_bit(&mut self, index: usize, bit: bool) {
```

Comment:
```suggestion
    pub fn set_bit(&mut self, index: usize, v: bool) {
```
(as called in other methods)

Reply:
fixed